homeassistant/components/hunterdouglas_powerview/entity.py

```diff
@@ -78,7 +78,8 @@ def device_info(self) -> DeviceInfo:
         )
 
         for shade in self._shade.shade_types:
-            if shade.shade_type == device_info[ATTR_MODEL]:
+            if str(shade.shade_type) == device_info[ATTR_MODEL]:
+                device_info[ATTR_HW_VERSION] = shade.shade_type
```

Comment:
Hardware version is expected to be the hardware revision of the device/chip model/etc.

Reply:
removed